appcues/src/main/java/com/appcues/data/model/ExperiencePrimitive.kt

```diff
@@ -21,17 +21,29 @@ internal sealed class ExperiencePrimitive(
     open val style: ComponentStyle,
 ) {
 
+    abstract val textDescription: String?
```

Comment:
this is all just matching IOS textDescription [implementation](https://github.com/appcues/appcues-ios-sdk/pull/252/files#diff-f378689d5b13c1f1ba657cc55252ad13715ba708cdec91ee38602b69deafe5bcR16)